settings.py

```diff
@@ -496,6 +496,9 @@
     ]
 }
 
+# Note: if you change this, make sure to update the cronjob in commcarehq-ansible
+RESTORE_PAYLOAD_DIR = os.path.join(FILEPATH, 'restore_payloads')
```

Comment:
Ok, took this https://github.com/dimagi/commcare-hq/commit/96d8380783b66b0b2af500347fa4b1e0d0dd1dc4 from the touchforms example. This should be all the change that's needed for this PR. I have to modify the ansible script to make the appropriate directory and set the appropriate variable